BookingsAPI/Bookings.API/Controllers/HearingsController.cs

```diff
@@ -153,6 +153,13 @@ public async Task<IActionResult> BookNewHearing(BookNewHearingRequest request)
 
             var cases = request.Cases.Select(x => new Case(x.Number, x.Name)).ToList();
 
+            var isJudgePresent = newParticipants.Any(x => x.HearingRole.Name.Equals("Judge"));
```

Comment:
This would belong better in the existing BookNewHearingRequestValidation since it's to determine if the payload is correct